doc/sintaxis.md

```diff
@@ -0,0 +1,5 @@
+## Comprobación de sintaxis
+
+En Go se suele utilizar por defecto la herramienta gofmt, que justamente realiza la función que estamos buscando solventar. 
```

Comment:
`gofmt`, como su propio nombre indica, formatea, y hace un parsing sintáctico, pero realmente no comprueba si compila o no. Como le he recomendado a @pluque01, [mira esta respuesta en StackOverflow](https://stackoverflow.com/questions/16863014/is-there-a-command-line-tool-in-golang-to-only-check-syntax-of-my-source-code) con las diferentes opciones.